### ISSUE TITLE
Looks like it does not work with alfred 3

### ISSUE THREAD

**Issue by obukhow** [jason0x43/alfred-redmine#3]
When I run command "rm login" I receive notification "Error: Exit status 1"

```
[2016-07-07 17:42:00][ERROR: action.script] 2016/07/07 17:42:00 Using config file /Users/roomine/Library/Application Support/Alfred 2/Workflow Data/com.jason0x43.alfred-redmine/config.json
2016/07/07 17:42:00 Using cache file /Users/roomine/Library/Caches/com.runningwithcrayons.Alfred-2/Workflow Data/com.jason0x43.alfred-redmine/cache.json
2016/07/07 17:42:00 args: []string{"./alfred-redmine", "do", "login"}
2016/07/07 17:42:00 op: 'do'
2016/07/07 17:42:00 kind: 'login'
2016/07/07 17:42:00 query: 
```

**Comment by jason0x43** [comment]
Hmm...I must still have Alfred 2 installed. I keep missing these issues. I'll look into it.

**Comment by jason0x43** [comment]
Ah, no, the issue is that I was already logged in, and the issue, which is version-related, doesn't come up unless you haven't logged in. I have a fix, I just need to update this workflow.

**Comment by jason0x43** [comment]
Fixed in [release 2.0.0](https://github.com/jason0x43/alfred-redmine/releases/tag/v2.0.0)
